### PR TITLE
name logic added

### PR DESCRIPTION
**Pull request by mashaBurdo**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1 +1,2 @@
-print('hello')
\ No newline at end of file
+name = input()
+print('hello', name, '!!!!!')
```
Comment (mashaBurdo) [inline main.py:2]: Add string fomating here please

Comment (tolstik94) [reply main.py]: Hello

Comment (tolstik94) [reply main.py]: как у вас дела ?

Comment (AksenovEvgeny) [inline main.py]: nothing is clear, but very interesting)))

File: main.py [hunk 2]
```diff
@@ -1 +1,7 @@
...
+print(1)
```
Comment (IvanisovBoris) [inline main.py:3]: Better with a comma

Comment (ViDov291088) [inline main.py:3]: all normal

Comment (SergeyShibut) [inline main.py:3]: Прикольно

File: main.py [hunk 3]
```diff
@@ -1 +1,7 @@
...
+print(2)
```
Comment (RomanYarosh) [inline main.py:4]: what's up bro

File: main.py (hunk 2 above)
Comment (tolstik94) [inline main.py:3]: wadhhwdhwadhdahjdwahjwdhjwadhjkawd

File: main.py [hunk 4]
```diff
@@ -1 +1,7 @@
...
+print(3)
+print(4)
```
Comment (T1fle) [inline main.py:6]: It is pretty cool, Marya

File: main.py (hunk 2 above)
Comment (mashaBurdo) [reply main.py:3]: you are right, thanx

Comment (ViDov291088) [reply main.py:3]: Здравствуйте синьор))

Comment (AksenovEvgeny) [inline main.py:3]: nothing is clear, but very interesting)))

File: main.py [hunk 5]
```diff
@@ -1 +1,7 @@
-print('hello')
\ No newline at end of file
+name = input()
+print('hello', name, '!!!!!')
```
Comment (375296677745) [inline main.py:2]: that's right

**Pull request merged**